src/Relude/Extra/Type.hs

```diff
@@ -1,18 +1,27 @@
 {-# LANGUAGE AllowAmbiguousTypes #-}
 {-# LANGUAGE CPP                 #-}
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE PolyKinds           #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications    #-}
+{-# LANGUAGE TypeFamilies        #-}
+{-# LANGUAGE TypeOperators       #-}
 
 {- |
 Copyright:  (c) 2018-2019 Kowainik
+                2019 TheMatten
```

Comment:
The code itself is good. But I need to ask you to not add your personal copyright notice to this file so Kowainik can keep ownership of all code. When everything is licensed under `Kowainik` name, then it's much easier to resolve various legal issues that can arise in future. For example, if we keep this copyright notice and decide to change the license in future, we will need to ask your personal agreement to do that. This seems like a high price to pay for implementing our proposal to add several type-level functions :slightly_smiling_face:

Reply:
Wasn't really sure about this and hoped you will sort that out - I don't care about these functions at all :D

Reply:
Done - will probably ask instead in the future when not sure :slightly_smiling_face:

Reply:
Thanks for understanding! And no worries 🙂